### PR TITLE
pass usernames directly to filter

### PR DESCRIPTION
**Pull request by snopoke**
## Summary
Code used by ICDS dump command.

## Safety Assurance

- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I am certain that this PR will not introduce a regression for the reasons below

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/dump_reload/sql/filters.py [hunk 1]
```diff
@@ -37,21 +37,16 @@ def get_filters(self, domain_name):
 
 
 class UsernameFilter(DomainFilter):
-    def __init__(self, user_ids=None):
-        self.user_ids = user_ids
+    def __init__(self, usernames=None):
```
Comment (mkangia) [inline corehq/apps/dump_reload/sql/filters.py:40]: wont this break https://github.com/dimagi/commcare-hq/blob/cb9d1ebd9385082d4daf16a3e0c8e0c6fca3613d/corehq/apps/dump_reload/sql/dump.py#L102 ? I assume that that always passes IDs

Comment (snopoke) [reply corehq/apps/dump_reload/sql/filters.py:40]: no it won't break it since that doesn't pass in anything

**Review by mkangia — APPROVED** [review]
Code used by ICDS dump command.

Can you share a link to this command?

**Comment by snopoke** [comment]
@mkangia https://github.com/dimagi/commcare-icds/pull/157/files#diff-2299e47508434b48a9b377144efb0fe3e4f283b35ce6539b3b641a968b5b2251R131

**Comment by mkangia** [comment]
that's useful. thanks @snopoke

**Pull request merged**